cmd/launch.go

```diff
@@ -209,8 +214,13 @@ func launchInteractive(h *ec2helper.EC2Helper) {
 		}
 	}
 
-	// Launch the instance.
-	_, err = h.LaunchInstance(simpleConfig, detailedConfig, confirmation == cli.ResponseYes)
+	// Launch On-Demand or Spot instance based on capacity type
+	if simpleConfig.CapacityType == question.CapacityTypes.OnDemand {
+		_, err = h.LaunchInstance(simpleConfig, detailedConfig, confirmation == cli.ResponseYes)
+	} else {
+		err = LaunchSpotInstance(h, simpleConfig, detailedConfig, confirmation)
```

Comment:
Should `LaunchSpotInstance()` be a receiver function, like `LaunchInstance()` is?

Reply:
Receiver functions can only be made inside the package where the struct is defined. Should I move the methods over to the ec2helper file like the LaunchInstance method?

Reply:
I think that makes sense, as ec2helper deals with EC2 API-related things, and `LaunchSpotInstance` will definitely be that